[NNC] Move computeNOperand operations over

Chillee opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #56718 Added skeleton of python dispatch key
* #56766 [nnc] ported some more ops + added vectors to argvalue
* #56679 moved lowerings out of the TensorExprKernel and into independent functions
* #56456 Added matmul/unified dtypes
* **#55372 [NNC] Move computeNOperand operations over**

Differential Revision: [D27884601](https://our.internmc.facebook.com/intern/diff/D27884601)

Chillee committed:
moved more lowerings over

[ghstack-poisoned]

Chillee committed:
Update on "moved more lowerings over"




[ghstack-poisoned]

Chillee committed:
Update on "moved more lowerings over"




[ghstack-poisoned]

Chillee committed:
Update on "moved more lowerings over"




[ghstack-poisoned]

Chillee committed:
Update on "moved more lowerings over"




[ghstack-poisoned]

Chillee committed:
Update on "moved more lowerings over"




[ghstack-poisoned]

Chillee committed:
Update on "moved more lowerings over"




[ghstack-poisoned]

Chillee committed:
Update on "moved more lowerings over"




[ghstack-poisoned]

Chillee committed:
Update on "moved more lowerings over"




[ghstack-poisoned]

Chillee committed:
Update on "moved more lowerings over"




[ghstack-poisoned]

Chillee committed:
Update on "moved more lowerings over"




[ghstack-poisoned]

Chillee committed:
Update on "moved more lowerings over"




[ghstack-poisoned]

Chillee committed:
Update on "moved more lowerings over"




[ghstack-poisoned]

ZolotukhinM commented on torch/csrc/jit/tensorexpr/kernel.cpp:
```diff
@@ -10,6 +11,8 @@
 #include <torch/csrc/jit/tensorexpr/ir_simplifier.h>
 #include <torch/csrc/jit/tensorexpr/loopnest.h>
 #include <torch/csrc/jit/tensorexpr/operators/conv2d.h>
+#include <csignal>
+#include <iostream>
```
Are these headers needed?

ZolotukhinM reviewed (approved):
Looks good to me! Just make sure to clang format the code and check that all tests pass before landing :)

ZolotukhinM commented:
Also, please include more details in the commit message - it would be hard to tell what this commit affects with the current message.

Chillee committed:
Update on "[NNC] Move computeNOperand operations over"


Differential Revision: [D27884601](https://our.internmc.facebook.com/intern/diff/D27884601)

[ghstack-poisoned]

Chillee committed:
Update on "[NNC] Move computeNOperand operations over"


Differential Revision: [D27884601](https://our.internmc.facebook.com/intern/diff/D27884601)

[ghstack-poisoned]

Chillee committed:
Update on "[NNC] Move computeNOperand operations over"


Differential Revision: [D27884601](https://our.internmc.facebook.com/intern/diff/D27884601)

[ghstack-poisoned]

Chillee committed:
Update on "[NNC] Move computeNOperand operations over"


Differential Revision: [D27884601](https://our.internmc.facebook.com/intern/diff/D27884601)

[ghstack-poisoned]
